samples/subsys/usb/mass/src/main.c

```diff
@@ -50,6 +50,10 @@ USBD_DEFINE_MSC_LUN(NAND, "Zephyr", "FlashDisk", "0.00");
 USBD_DEFINE_MSC_LUN(SD, "Zephyr", "SD", "0.00");
 #endif
 
+#if CONFIG_DISK_DRIVER_MMC
+USBD_DEFINE_MSC_LUN(CONFIG_MMC_VOLUME_NAME, "Zephyr", "MMC", "0.00");
```

Comment:
You didn't check whether it works, did you? The problem is that USBD_DEFINE_MSC_LUN uses the first parameter via `STRINGIFY()` and therefore the disk functions will be called with `"SD"` instead of `SD`.